ui/client/containers/Visualization.js

```diff
@@ -35,24 +36,33 @@ class Visualization extends React.Component {
         this.showModalDetailsIfNeeded(details.fetchedProcessDetails.json);
         this.showCountsIfNeeded(details.fetchedProcessDetails.json);
       })
+
+      this.fetchProcessStatus()
+    }).catch((error) => {
```

Comment:
should we catch also errors from fetchProcessStatus, fetchProcessDefinition etc.?

Reply:
This is enought - when process throw 404 than this will not run.